Dockerfile

```diff
@@ -24,14 +24,18 @@ RUN \
     echo '' > requirements.txt && \
     pip3 install . --no-deps
 
-RUN pip install jsonformatter==0.3.1
+RUN pip install jsonformatter==0.3.2 gql[requests]==3.3.0
 
 RUN mkdir /var/log/freshmaker/
 
 RUN mkdir -p /usr/share/freshmaker && cp contrib/freshmaker.wsgi /usr/share/freshmaker/
 # Delete the default logging configuration
 RUN rm -f fedmsg.d/freshmaker-logging.py
 
+RUN mkdir /etc/freshmaker
+COPY ./conf/config.py /etc/freshmaker/config.py
+COPY ./conf/koji.conf /etc/koji.conf
```

Comment:
Freshmaker heavily relies on the internal infrastructure but this is a public repository, so we can't easily include our configurations in this repo, when we deploy freshmaker in OpenShift, the config files will be mounted through volumes, so we don't need to copy these config templates to container.

Reply:
I see! I removed these lines, thanks for the explanation!